packages/cli/src/commands/init.js

```diff
@@ -11,18 +11,18 @@ const register = program => program
   .command(signature, { noHelp: true })
   .usage('<project-name> [version]')
   .description(description)
-  .option('--full', 'create an application, package, and implementation directory contracts to manage your project')
+  .option('--publish', 'automatically publishes your application upon pushing it to a network')
```

Comment:
I'd use the same description you used for the `publish` command. Note that you are using `project` for the other and here you are using `application`